docs/src/documentation/03-components/01-action/button/01-guidelines.mdx

```diff
@@ -16,7 +16,7 @@ check out our [interactive guide on action components](/guides/using-button-type
 
 - To offer the most important actions users can take.
 
-### When not to use
+## When not to use
```

Comment:
I'm pretty sure all of these were intentional. 🤷  Because this can be treated as related to "When to use". Unless you're sure about this change, I wouldn't do it.

Reply:
Confirmed that this changed was requested.

Reply:
:crying_cat_face: A loss of semantics.